Task/Task-NonWGSRareDiseaseTestOrderCancelled-FollowupTest-Example.json

```diff
@@ -0,0 +1,57 @@
+{
+  "resourceType": "Task",
+  "id": "Task-NonWGSRareDiseaseTestOrderCancelled-FollowupTest-Example",
+  "basedOn": [
+    {
+      "reference": "ServiceRequest/ServiceRequest-NonWGSTestOrderForm-FollowupTestTypes-Example"
+    }
+  ],
+  "status": "cancelled",
+  "businessStatus": {
```

Comment:
Move to statusReason, maybe with code InvalidRequest? though it is unclear in the scenario why the original request has been closed

Reply:
Changed to: DecisionNotToProgressWithTestRequest and retained Task.code as Interpretation due to statement on Page17 (Follow Up Test to be requested to aid interpretation)